skin.json

```diff
@@ -146,6 +146,9 @@
       "height": 81
     }
   },
+  "live": {
+    "dvrEnabled": false
```

Comment:
In the buttons array, we have the "live" indicator entry. Doesn't removing that have the desired affect here? Or is DVR on mobile more complicated than desktop.